api/v1alpha1/common_types.go

```diff
@@ -5,3 +5,11 @@ type NamespacedName struct {
 	Name      string `json:"name"`
 	Namespace string `json:"namespace"`
 }
+
+// SlaSpec defines what is the percentage
+type SlaSpec struct {
```

Comment:
it's not an SLA

Reply:
I think that's not up to date yet, we talked about that one already 🙂